### PR TITLE
Add Symfony 6 support

### PR DESCRIPTION
**Pull request by AbdelilahJabri**

### CODE REVIEW

**Comment by AbdelilahJabri** [comment]
@tgalopin can you check this please?

**Review by hamzanatek — APPROVED** [review]

File: .github/workflows/test.yaml [hunk 1]
```diff
@@ -66,3 +66,35 @@ jobs:
                   composer require --no-update --dev symfony/framework-bundle=$SYMFONY_VERSION symfony/yaml=$SYMFONY_VERSION
                   composer update --prefer-dist --no-interaction --no-ansi --no-progress
                   php vendor/bin/simple-phpunit
+
+    tests-php-7-4-symfony-6-0:
```
Comment (tgalopin) [inline .github/workflows/test.yaml:70]: You can remove this case, Symfony 6 requires PHP 8 :)

**Review by tgalopin — COMMENTED** [review]
Thanks for the PR!

**Comment by gndk** [comment]
@AbdelilahJabri Thanks for getting the PR started! Do you have time to continue with this or should somebody else take over? Seems like only minor changes are required to get this merged after we see a green pipeline for `tests-php-8-0-symfony-6-0`.

**Comment by yoshz** [comment]
Any progress on this? 
Removing the php-7.4 job and restarting the tests should probably already fix the tests.

**Comment by tgalopin** [comment]
Thanks @AbdelilahJabri.

**Pull request merged**